umpf

```diff
@@ -263,6 +268,20 @@ setup() {
 			GIT_REMOTE="${1}"
 			shift
 			;;
+		--override)
+			IFS=: read -r topic commitish <<< "${1}"
+			if [ -z "${commitish}" ]; then
+				commitish="${topic}"
```

Comment:
The result here is imho pretty confusing: With `--identical` you're doing two things here at once:
 1. use the current head of the branch instead of the same commit as before
 2. use the _local_ branch instead of the one from the chosen remote.

I don't think, this is how it should work. Without the second argument, the current head of the remote branch should be used. That would be more consistent with how umpf works in general.

Reply:
- if we have `--remote=foo`, `--override topic` -> `--override topic:foo/topic`
- If we have no `--remote`, `--override topic` -> `--override topic:topic`

Ye, above makes sense.

Reply:
Fixed.